docs/docs/migration-guide.mdx

```diff
@@ -55,6 +55,15 @@ We've updated our docs to reflect these changes. All examples now show a direct
 correct spaCy model. We've also added a warning to the [SpaCyNLP](components.mdx#spacynlp)
 docs that explains the fallback behavior.
 
+### Forms
+
+#### New `ignored_intents` parameter in Forms
+
+We added a new parameter under Forms with the name `ignored_intents`. This parameter
+can be used in case the user never wants to fill any slots of a form with the specified
+intent, eg. chitchat. Please see the [Forms documentation](forms.mdx) for examples and more
+information on how to use it in your `domain.yml` file.
+
```

Comment:
```suggestion
#### New `ignored_intents` parameter in Forms

There is a new parameter under Forms called `ignored_intents`. This parameter
can be used to prevent any required slots in a form from being filled with the specified
intent or intents. Please see the [Forms documentation](forms.mdx) for examples and more
information on how to use it in your `domain.yml` file.

```